pkg/docker/index.html

```diff
@@ -346,22 +346,26 @@ <h3 translatable="yes">Additional Storage</h3>
         <input type="text" name="host" class="form-control" placeholder="{{placeholder}}">
       </div>
       <div class="form-group">
-        <div class="btn-group dropdown mount-mode" id="mount-mode-select">
+        <div class="btn-group dropdown mount-mode" id="mount-mode-select" data-value="">
           <button class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown">
             <span translatable="yes">Default</span>
             <div class="caret"></div>
           </button>
           <ul class="dropdown-menu">
-            <li><a translatable="yes" data-value="default">Default</a></li>
-            <li><a translatable="yes" data-value="read-write">ReadWrite</a></li>
+            <li><a translatable="yes" data-value="">Default</a></li>
+            {{^selinux}}
```

Comment:
Do we really want to use inverted section here? I thought that in the case that selinux is actually available we still want  the option to not have the selinux flags in the mount command.

Reply:
@garrett Would you be able to make a call? The dropdown menu gets very large with all the options.

@KKoukiou Would you be able to take a screenshot of the full menu with all Linux options and attach it? (ie: without my changes)